modules/databases/mysql_flexible_server/server.tf

```diff
@@ -15,7 +15,7 @@ resource "azurerm_mysql_flexible_server" "mysql" {
   location            = var.location
   version             = try(var.settings.version, null)
   sku_name            = try(var.settings.sku_name, null)
-  zone                = try(var.settings.zone, null)
+  zone                = try(var.settings.zone, 1)
```

Comment:
Waarom is dit nu 1? Is dit een ok default value? Of is null goed zodat hij weggelaten word indien niet gezet?

Reply:
is een bug. als je geen HA opgeeft default hij op 1. Gaat bij eerste deploy goed. Daarna probeert hij er null van te maken